### DISCUSSION TITLE
iAlarm XR (Antifurto365) e Home Assistant

### DISCUSSION THREAD

**Discussion by Salsalove** [indomus/forum#1293] [HUB: Home Assistant]
Buongiorno, sono in procinto di acquistare un kit iAlarm XR-5 ma solo da poco ho letto che l'integrazione ufficiale di Antifurto365 non supporta i nuovi modelli XR. Mi confermate che è così?
Per ovviare al "problema" ho letto di ialarm-mqtt...funziona bene? E soprattutto si possono leggere gli stati di tutti i sensori di movimento, sensori porte/finestre, sirene, sensori perimetrali ecc e soprattutto inserire/disinserire l'allarme con codice? 

Se qualcuno può darmi consigli ne sarei grato.

**Comment by snakuzzo** [comment]
Hai fatto una ricerca tra le discuzzioni ?

Guatda [QUI](https://github.com/indomus/forum/discussions?discussions_q=ialarm)
